video_xblock/video_xblock.py

```diff
@@ -614,6 +621,41 @@ def download_transcript(self, request, suffix=''):  # pylint: disable=unused-arg
         response.headerlist = headerlist
         return response
 
+    @XBlock.json_handler
+    def dispatch(self, request, suffix):
+        """
+        Dispatch request to XBlock's player.
+        Args:
+            request: incoming request data.
+            suffix: slug used for routing.
+        """
+        return self.get_player().dispatch(request, suffix)
+
+    @XBlock.handler
+    def ui_dispatch(self, _request, suffix):
```

Comment:
Because on the one hand, I must follow `XBlock.handler` interface. On the other, I don't use `request` inside the function.

Hence pylint will complain about an unused argument. I can either put `#pylint: disable=unused-argument` or make it `_request` to explicitly mark argument as unused.